src/app/(infoscreen)/@restaurants/page.tsx

```diff
@@ -0,0 +1,39 @@
+import { Slide } from '@/components/Carousel';
+import { RestaurantMenu } from '@/components/Restaurants/Restaurant';
+import { getRestaurants } from '@/server/restaurants';
+
+const Restaurants = async () => {
+	const restaurants = await getRestaurants([
+		'Chemicum',
+		'Exactum',
+		'Kaivopiha',
+	]);
+
+	console.dir(Object.values(restaurants), { depth: 3 });
+
+	return (
+		<Slide full className="bg-green-900 pb-0">
+			{Object.values(restaurants).length === 0 ? (
+				<p>There are no open restaurants at the moment.</p>
```

Comment:
old system has this state as the lofi girl gif

Reply:
Would rather leave more view time for other slides if none are open, setting this to null instead